src/tools/dev_server/hot-reload-server.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * @license
+ * Copyright (c) 2019 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import WebSocket from 'ws';
+import chokidar from 'chokidar';
+
+export class HotReloadServer {
+    private server: WebSocket.Server | null;
```

Comment:
this files appears to be using 4 space indent instead of the usual 2 space indent we use. Can you please change it to 2 spaces?

What probably happened is VSCode adapts to the file, but it uses 4 spaces by default - so you may want to change the default.

Reply:
Fixed! :)